metadata-ingestion/tests/unit/config/test_pydantic_validators.py

```diff
@@ -92,11 +97,38 @@ class TestModel(ConfigModel):
     v = TestModel.parse_obj({"b": "original"})
     assert v.b == "original"
 
-    v = TestModel.parse_obj({"b": "original", "d1": "deprecated", "d2": "deprecated"})
+    with pytest.warns(ConfigurationWarning, match=r"d\d.+ deprecated"):
+        v = TestModel.parse_obj(
+            {"b": "original", "d1": "deprecated", "d2": "deprecated"}
+        )
     assert v.b == "original"
     assert v.d1 == "deprecated"
     assert v.d2 == "deprecated"
     assert any(["d1 is deprecated" in warning for warning in get_global_warnings()])
     assert any(["d2 is deprecated" in warning for warning in get_global_warnings()])
 
     clear_global_warnings()
+
+
+def test_multiline_string_fixer():
```

Comment:
Can you add a test with a dash in the secret name?

Reply:
that will come in a follow up PR - this one doesn't fix that just yet